windows/deploy_windows.ps1

```diff
@@ -282,10 +270,21 @@ Function Build-Installer
         }
     }
 
-    Invoke-Native-Command -Command "$WindowsPath\NSIS\makensis" `
-        -Arguments ("/v4", "/DAPP_NAME=$AppName", "/DAPP_VERSION=$AppVersion", `
-        "/DROOT_PATH=$RootPath", "/DWINDOWS_PATH=$WindowsPath", "/DDEPLOY_PATH=$DeployPath", `
-        "$WindowsPath\$InstallerNSI")
+    if ($BuildOption -ne "")
+    {
+        Invoke-Native-Command -Command "$WindowsPath\NSIS\makensis" `
+            -Arguments ("/v4", "/DAPP_NAME=$AppName", "/DAPP_VERSION=$AppVersion", `
+            "/DROOT_PATH=$RootPath", "/DWINDOWS_PATH=$WindowsPath", "/DDEPLOY_PATH=$DeployPath", `
+            "/DBUILD_OPTION=$BuildOption", `
```

Comment:
Can't we pass an empty BUILD_OPTION? This would avoid the if here.

Reply:
I tried that, but when the command is invoked, it generates a script error. So we probable would end up having another "if" somewhere else to cover that.

Reply:
Hm, maybe there's still a way to avoid duplicating this rather long `Invoke-Native-Command` call?

Reply:
I guess you could put the `-Arguments` into a list, apart from `/DBUILD_OPTION`, then - if `$BuildOption` is set, insert an extra argument into the list in the last but one position, then pass th list of arguments to `-Arguments`.